drivers/RawSerial.h

```diff
@@ -58,10 +58,8 @@ namespace mbed {
  * @endcode
  */
 class
-    MBED_DEPRECATED_SINCE(
-        "mbed-os-6.0.0",
-        "Use UnbufferedSerial instead."
-    ) RawSerial: public SerialBase, private NonCopyable<RawSerial> {
+    MBED_DEPRECATED("The class has been deprecated and will be removed in the future. Use UnbufferedSerial instead.")
```

Comment:
why are we removing `since`? I find it good information to have

Reply:
UARTSerial is not available with Mbed OS 6. I find the original print confusing.